.classpath

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The .classpath should not be pushed

Reply:
added the file name in .gitignore

Reply:
It is still here.

Reply:
removed from repository in commit number fa9ddf7